projects/batfish-client/src/org/batfish/client/Client.java

```diff
@@ -315,6 +325,176 @@ private boolean answer(String questionTemplateName, String paramsLine,
+         case BOOLEAN:
+            if (!value.isBoolean()) {
+               throw new BatfishException(errorMessage);
+            }
+            break;
+         case COMPARATOR:
+            if (!(COMPARATORS.contains(value.textValue()))) {
+               throw new BatfishException(errorMessage);
+            }
+            break;
+         case INTEGER:
+            if (!value.isInt()) {
+               throw new BatfishException(errorMessage);
+            }
+            break;
+         case IP:
+            if (!value.isTextual()) {
+               throw new BatfishException(errorMessage);
+            }
+            // TODO: Need to double check isInetAddress()
+            if (!InetAddresses.isInetAddress(value.textValue())) {
+               throw new BatfishException(
+                     String.format("The input: %s is not a valid ip address.",
+                           value.textValue()));
+            }
+            break;
+         case IP_WILDCARD:
+            if (!value.isTextual()) {
+               throw new BatfishException(errorMessage);
+            }
+            // TODO: need ip_WildCard validation check
+            break;
+         case JAVA_REGEX:
+            if (!value.isTextual()) {
+               throw new BatfishException(errorMessage);
+            }
+            try {
+               Pattern.compile(value.textValue());
+            } catch (PatternSyntaxException e) {
+               throw new BatfishException(
+                     String.format("Input String: %s is not a valid Java " +
+                           "regular  expression for variable type: %s.",
+                           value.textValue(), expectedType));
+            }
+            break;
+         case JSON_PATH_REGEX:
+            if (!value.isTextual()) {
+               throw new BatfishException(errorMessage);
+            }
+            validateJsonPathRegex(value.textValue());
+            break;
+         case PREFIX:
+            if (!value.isTextual()) {
+               throw new BatfishException(errorMessage);
+            }
+            // TODO: need prefix validation check
+            break;
+         case PREFIX_RANGE:
+            if (!value.isTextual()) {
+               throw new BatfishException(errorMessage);
+            }
+            // TODO: need prefix range validation check
+            break;
+         case STRING:
+            if (!value.isTextual()) {
+               throw new BatfishException(errorMessage);
+            }
+            break;
+         case SUBRANGE:
+            if (!value.isTextual()) {
+               throw new BatfishException(errorMessage);
+            }
+            // TODO: need prefix range validation check
+            break;
+         case JSON_PATH:
+            validateJsonPath(value);
+            break;
+      }
+   }
+
+   /**
+    * Validate that {@code jsonPathRegex} contains a valid Java regular
+    * expression of a JsonPath(Starts with '/', ends with either '/' or
```

Comment:
use proper formatting.

1. Space between `JsonPath` and `(`.
2. Use `{@code JsonPath}`, for example.
3. Use `"` for Java strings, as `'/i'` is not a legal Java token.
4. Newline before throws
5. Capitalize Java.